externals/kyuubi-jdbc-engine/src/main/resources/META-INF/services/org.apache.kyuubi.engine.jdbc.connection.JdbcConnectionProvider

```diff
@@ -16,4 +16,5 @@
 #
 
 org.apache.kyuubi.engine.jdbc.doris.DorisConnectionProvider
-org.apache.kyuubi.engine.jdbc.phoenix.PhoenixConnectionProvider
\ No newline at end of file
+org.apache.kyuubi.engine.jdbc.phoenix.PhoenixConnectionProvider
+org.apache.kyuubi.engine.jdbc.postgresql.PostgreSqlConnectionProvider
```

Comment:
let's add an additional newline at the end of the file to suppress the GitHub style complaint

Reply:
update